alonzo/impl/src/Cardano/Ledger/Alonzo.hs

```diff
@@ -56,8 +59,10 @@ instance
   (CC.Crypto c) =>
   Shelley.ValidateScript (AlonzoEra c)
   where
+  scriptPrefixTag _proxy = "\x00"
   validateScript = error "TODO: implement validateScript"
-  hashScript = error "TODO: implement hashScript"
+
+-- use the default method for hashScript
```

Comment:
rogue comment?

Reply:
Just commenting that there is no method definitition for hashScript since we are using the default method. I made this clearer.